maltd-frontend/src/__snapshots__/storyshots.test.js.snap

```diff
@@ -256,6 +256,30 @@ exports[`Storyshots List Element user 1`] = `
 </div>
 `;
 
+exports[`Storyshots List Element with resources 1`] = `
+<div
+  className="project-div"
+>
+  <div
+    className="project-list-item"
+    id="project-info"
+  >
+    <strong
+      className="large-size"
+    >
+      MALTD
+    </strong>
+    <p
+      className="project-list-item"
+      id="member-resources"
+    >
+      Dyn, Share
+    </p>
+  </div>
+  <div />
```

Comment:
Couldn't tell if this is an orphan div tag or not. Is this intentional?

Reply:
hmm not sure cause this is just in the storyshot snapshot - I checked the actual `ListElement.js` file and there are no orphan tags so this should be ok as is